runtime/mainnet/src/lib.rs

```diff
@@ -1040,4 +1132,198 @@ mod tests {
 
 		assert_eq!(deposit(2, 64), system_para_deposit(2, 64))
 	}
+
+	#[test]
+	fn treasury_account_is_pallet_id_truncated() {
+		assert_eq!(TreasuryAccount::get(), TREASURY_PALLET_ID.into_account_truncating());
+	}
+
+	const ALICE: [u8; 32] = [1; 32];
+	pub fn new_test_ext() -> sp_io::TestExternalities {
+		let initial_balance = 100_000_000 * UNIT;
+		let mut t = frame_system::GenesisConfig::<Runtime>::default().build_storage().unwrap();
+		pallet_balances::GenesisConfig::<Runtime> {
+			balances: vec![
+				(TreasuryAccount::get(), initial_balance),
+				(MaintenanceAccount::get(), initial_balance),
+				(ALICE.into(), initial_balance),
+			],
+		}
+		.assimilate_storage(&mut t)
+		.unwrap();
+		let mut ext = sp_io::TestExternalities::new(t);
+		ext.execute_with(|| System::set_block_number(1));
+		ext
+	}
+
+	#[test]
+	fn transaction_payment_charges_fees_via_balances_and_funds_treasury_and_maintenance_equally() {
+		new_test_ext().execute_with(|| {
+			let who: AccountId = ALICE.into();
+			let call = RuntimeCall::System(frame_system::Call::remark { remark: vec![] });
+			let fee = UNIT / 10;
+			let tip = UNIT / 2;
+			let fee_plus_tip = fee + tip;
+			let treasury_balance =
+				pallet_balances::Pallet::<Runtime>::free_balance(&TreasuryAccount::get());
```

Comment:
Using Balances::free_balance can greatly improve the readability of tests, especially if there is wrapping.

Any Pallet<Runtime> should be replaced with the runtime alias. It should result in much cleaner, simpler code with minimal effort.

Reply:
https://github.com/r0gue-io/pop-node/commit/5b6f7315528a1dfd987eb5bfa2abd01949e8cf5a